2025-06-16 03:21:09,304	INFO worker.py:1917 -- Started a local Ray instance.
[2m[36m(pid=494088) [0m 0:   0%|          | 0.00/15.0 [00:00<?, ?it/s]                                                            [2m[36m(pid=494088) [0m 0:   7%|â–‹         | 1.00/15.0 [00:00<00:00, 101it/s][2m[36m(pid=494088) [0m 0:  60%|â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ    | 9.00/15.0 [00:01<00:00, 8.52it/s][2m[36m(pid=494088) [0m 0:  60%|â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ    | 9.00/15.0 [00:01<00:00, 8.52it/s][2m[36m(pid=494088) [0m 0: 16.0it [00:02, 6.99it/s]                          [2m[36m(pid=494088) [0m 0: 16.0it [00:02, 6.99it/s][2m[36m(pid=494088) [0m 0: 24.0it [00:03, 6.88it/s][2m[36m(pid=494088) [0m 0: 24.0it [00:03, 6.88it/s][2m[36m(pid=494088) [0m 0: 30.0it [00:04, 5.88it/s][2m[36m(pid=494088) [0m 0: 30.0it [00:04, 5.88it/s]                                         [2m[36m(pid=494088) [0m 0: 30.0it [00:04, 6.35it/s]
[Jun 16, 2025 03:21:35AM] No errors!
